go/libraries/doltcore/env/actions/branch.go

```diff
@@ -160,7 +160,7 @@ func DeleteBranchOnDB(ctx context.Context, ddb *doltdb.DoltDB, dref ref.DoltRef,
 		}
 
 		isMerged, _ := master.CanFastReverseTo(ctx, cm)
-		if err != nil && err != doltdb.ErrUpToDate {
+		if err != nil && errors.Is(err, doltdb.ErrUpToDate) {
```

Comment:
You inverted the logic here, purposeful?

Reply:
`func Is(err error, target error) bool`

Reply:
no, I mean the previous version had !=, and you changed it to == (effectively)